backend/src/lib/repositories/homepage-repo.ts

```
@@ -71,11 +72,12 @@ class HomepageRepository extends BaseRepository {
       });
     } catch (error) {
       if (error.code === "ConditionalCheckFailedException") {
-        console.log("Someone else updated the item before us");
-        return;
-      } else {
-        throw error;
+        logger.warn(
+          "ConditionalCheckFailed on update homepage=%s. Someone else updated the homepage before us",
+          title
+        );
       }
+      throw error;
```

Should we throw an error, we didn't before.